api/src/services/export/feedback-mapper.js

```diff
@@ -0,0 +1,49 @@
+const moment = require('moment');
```

Comment:
There should be a test (unit or otherwise, you know me) that covers this logic

Reply:
You're right - I'm ashamed...